### DISCUSSION TITLE
Looking for data on new Covid patients in ICUs per day, or length of stay

### DISCUSSION THREAD

**Discussion by denis-bz** [marlon360/rki-covid-api#413] [Q&A]
Just a question,
does anyone know of data on new Covid patients in ICUs per day / per week,
to estimate how long they stay ? 
Length-of-stay would seem to me useful in estimating load on the hospital system --
a very few patients who stay a long time can clog the system
(just as a few slow trucks clog the Autobahn).
I see papers from 2020 - 2021  but so far no downloadable data :/

Length-of-stay example:
    2100 ICU beds occupied with Covid patients
    suppose 300 patients arrive, 300 leave each day
    => average length-of-stay *roughly* 7 days
    
Also, does anyone know of a summary of which RKI numbers on Hospitalisierungsrate, 7-Tage Inzidenz etc.
are for all hospitals, which for ICUs only ?
find this confusing, hard to compare with intensivregister.de .

Thanks,
cheers
    -- denis

<sub> In God we trust, all others bring data </sub>

**Comment by Rubber1Duck** [comment]
I convert this to a discussion ...............
